cli/src/common_utils/utility_scripts.sh

```diff
@@ -133,3 +133,21 @@ function ensure_healthy_cluster {
 
     _linfo "*****************************************************************"
 }
+function set_lrserial_number {
```

Comment:
into a separate file:
./cli/src/common_utils/lr-serial-number

Let this script echo lr-serial-number from file if no arg is provided, else it sets the lr-serial-number, just like hostname command.